cookbook/webpack-encore.rst

```diff
@@ -44,10 +49,23 @@ use Webpack Encore with Sulu, some configuration has to be adjusted,
 because there is an additional Javascript application for Sulu’s admin interface.
 
 To continue, create a directory ``assets/website/`` and move the
-following directories into that directory:
+following files into that directory:
 
-* ``assets/css/``
-* ``assets/js/``
+* ``assets/styles``
+* ``assets/app.js``
+
+As long as the recipes installs the ``stimolus`` library we have to remove the following files / directories:
```

Comment:
webpack encore is not related to sulu, so I would not make suggestions on what to use of webpack encore and what to throw away. I would just tell the user to move **all** new files in the `assets/` directory to the `assets/website/` directory.

Reply:
From this docs point it make sense to move the stimulus things just into the assets/website folder. From web-js point it make sense to not have this.

Reply:
We always say, sulu is frontend independent, so we also cannot make an exception in favor of web-js IMO.
But it would be okay for me to have an additional section describing how to remove stimolus and replace it with web-js.

Reply:
But it should be clearly pointed out, that this section is fully optional and it's definitely not required to replace stimolus with web-js to use webpack encore.